internal/nexodus/keys_darwin.go

```diff
@@ -0,0 +1,25 @@
+//go:build darwin
+
+package nexodus
+
+import "fmt"
+
+// handleKeys will look for an existing key pair, if a pair is not found this method
+// will generate a new pair and write them to location on the disk depending on the OS
+func (ax *Nexodus) handleKeys() error {
```

Comment:
We should add moving to https://pkg.go.dev/golang.zx2c4.com/wireguard/wgctrl/wgtypes#GenerateKey to the backlog